android/assets/jsons/Civ V - Vanilla/Nations.json

```diff
@@ -804,7 +804,7 @@
 		"name": "Byzantium",
 		"leaderName": "Theodora",
 		"adjective": ["Byzantine"],
-		"startBias": ["Coats"],
```

Comment:
Maybe they just happen to really like coats. Fair enough.

Reply:
Calls for a mod where Byzantium always spawns near a Natural Wonder named "Coat Rack" and gets a unique Worker replacement named "Coats" and a Warrior replacement named "Coåts" and an Archer replacement named "Cøats" and a Scout replacement named "Çoats" - all having the exact same icon and pixel unit graphic... and so on...